apps/AEPSampleApp/ios/AEPSampleApp/AppDelegate.m

```diff
@@ -64,8 +64,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   [AEPMobileCore setLogLevel: AEPLogLevelTrace];
   [AEPMobileCore registerExtensions: @[AEPMobileEdgeIdentity.class,
                                        AEPMobileEdge.class,
-                                       AEPMobileMessaging.class,
-                                     AEPMobileOptimize.class] completion:^{
```

Comment:
Should AEPMobileOptimize be removed here?

Reply:
The ci fails when I remove this, it is based in staging branch.  I have to leave  the AEPMobileOptimize here.  Planning  to merge this PR after Optimize PR, so should not be a problem.

Reply:
Just to confirm we are removing Optimize here and it will be added back with the Optimize PR?

Reply:
I was able to remove the Optimize extension without CI failure.  Will keep it this way for edgeIdentity branch.  It will be added back once merge to staging with AJO extension PR changes.